terraform/modules/hub/hub_frontend.tf

```diff
@@ -48,98 +48,98 @@ locals {
   }
```

Comment:
Note: terraform 0.12 correctly dedents this heredoc, where terraform 0.11 didn't.  This causes a rebuild of the frontend task definition.  This is ok though and is generally a change for the better.